<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTPGUI</class>
 <widget class="QMainWindow" name="FTPGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTPGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cmd_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Command Prompt:</string>
    </property>
   </widget>
   <widget class="QLabel" name="localdir_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>112</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Local work directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="localdir_edit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="remotedir_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>154</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote work directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remotedir_edit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ldir_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rdir_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="list_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>List:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="list_btn">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>230</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QListWidget" name="remote_list">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>259</width>
      <height>441</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="cmd_list">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>259</width>
      <height>441</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="file_cbox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>rename</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>get</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>put</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>clear</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="ldir_cbox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>get</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>clear</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="rdir_cbox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>get</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mk</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>del</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>list</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>clear</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="mode_cbox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>pasv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>port</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>720</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IP_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="port_label">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>30</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string> Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_edit">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>30</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="IP_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>73</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>IP/domain:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="con_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Pwd:</string>
    </property>
   </widget>
   <widget class="QLabel" name="usr_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>27</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Usr:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwd_edit">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>70</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>400</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="usr_edit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename2_edit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="file_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="to_label">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>190</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename4_edit">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>190</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="from_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string> from:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
